infrastructure/helm/quantumserverless/values.yaml

```diff
@@ -97,43 +85,23 @@ rayClusterEnable: true
 ray-cluster:
   nameOverride: "kuberay"
   fullnameOverride: ""
-
   image:
     repository: "qiskit/quantum-serverless-ray-node"
     tag: "nightly-py39"
     pullPolicy: IfNotPresent
-
   head:
     rayStartParams:
-      dashboard-host: '0.0.0.0'
-    ports: [{containerPort: 10001, name: client},{containerPort: 6379, name: redis},{containerPort: 8265, name: dashboard},{containerPort: 8080, name: metrics},{containerPort: 8000, name: serve},{containerPort: 4180, name: proxy}]
-    sidecarContainers:
-        - name: oauth-proxy
-          image: quay.io/oauth2-proxy/oauth2-proxy:v7.3.0
-          imagePullPolicy: IfNotPresent
-          args:
-          - --client-secret=CLIENTSECRET-CHANGEME
-          - --oidc-issuer-url=http://LOCAL-IP:31059/realms/quantumserverless
-          - --oidc-extra-audience="account"
-          - --email-domain="*"
-          - --insecure-oidc-allow-unverified-email=true
-          - --http-address=0.0.0.0:4180
-          - --cookie-secret=SECRET0123456789
-          - --provider=keycloak-oidc
-          - --client-id=rayclient
-          - --upstream="http://HELM-RELEASE-kuberay-head-svc:8265"
-          - --redirect-url=http://localhost/oauth2/callback
+      dashboard-host: "0.0.0.0"
+    ports: [{containerPort: 8265, name: dashboard}]
```

Comment:
How the `gateway` will be the `entrypoint` for the project I started to remove the `sidecarContainers` from other services.

Reply:
The port **8265** is used by the Ray console as well as the APIs accessed through the gateway.  This side car probably still is needed.